logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/GetMessageAssetUseCase.kt

```diff
@@ -72,7 +74,10 @@ internal class GetMessageAssetUseCaseImpl(
             CompletableDeferred(MessageAssetResult.Failure(it))
         }, { message ->
             val assetDownloadStatus = (message.content as MessageContent.Asset).value.downloadStatus
+            val assetUploadStatus = (message.content as MessageContent.Asset).value.uploadStatus
```

Comment:
maybe create variable to not cast twice

Reply:
Right, I even changed the order and started with `when`, because we don't allow any content other than `MessageContent.Asset` here